steering_controllers_library/doc/userdoc.rst

```diff
@@ -0,0 +1,88 @@
+.. _steering_controllers_library_userdoc:
+
+steering_controllers_library
+=============================
+
+Library with shared functionalities for mobile robot controllers with steering drive (2 degrees of freedom).
+The library implements generic odometry and update methods and defines the main interfaces.
+
+Nomenclature used for the controller is used from `wikipedia <https://en.wikipedia.org/wiki/Wheelbase>`_.
+
+Execution logic of the controller
+----------------------------------
+
+The controller uses velocity input, i.e., stamped or unstamped Twist messages where linear ``x`` and angular ``z`` components are used.
+Angular component under
+Values in other components are ignored.
+In the chain mode the controller provides two reference interfaces, one for linear velocity and one for steering angle position.
+Other relevant features are:
+
+  - support for front and rear steering configurations;
+  - odometry publishing as Odometry and TF message;
+  - input command timeout based on a parameter.
```

Comment:
```suggestion
* support for front and rear steering configurations;
* odometry publishing as Odometry and TF message;
* input command timeout based on a parameter.
```

I updated the template from control.ros.org, now I suggest using an asterisk without spaces in front, e.g., see
https://github.com/ros-controls/ros2_controllers/blob/6851df137cf2118d1a7166921d34eb759c225526/joint_trajectory_controller/doc/userdoc.rst?plain=1#L34